packages/expo-notifications/src/Notifications.types.ts

```diff
@@ -768,6 +768,10 @@ export type MaybeNotificationResponse = NotificationResponse | null | undefined;
 export {
   PermissionExpiration,
   PermissionResponse,
+  /**
+   * @deprecated use the `Subscription` type instead
+   * */
```

Comment:
I wouldn't be surprised if TS doesn't recognize this doc comment (inside `export {}`). Have you checked if this appears in the docs?

Reply:
it now renders as expected